sphinx/templates/quickstart/conf.py_t

```diff
@@ -141,7 +141,13 @@ latex_documents = [
     (master_doc, '{{ project_fn }}.tex', u'{{ project_doc_texescaped_str }}',
      u'{{ author_texescaped_str }}', 'manual'),
 ]
-
+{% if language|default('').split('_')[0] == 'zh' %}
```

Comment:
Jinja2 raises TemplateSyntaxError in this line:
```
TemplateSyntaxError: expected token 'end of statement block', got '.'
```
https://travis-ci.org/sphinx-doc/sphinx/jobs/184147701#L2019

Does this really work correct in your environment?